src/SFA.DAS.AANHub.Application/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandHandler.cs

```diff
@@ -0,0 +1,79 @@
+﻿using MediatR;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Models;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.AANHub.Domain.Interfaces;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
+
+namespace SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember
+{
+    public class PatchApprenticeMemberCommandHandler : IRequestHandler<PatchApprenticeMemberCommand, ValidatableResponse<PatchApprenticeMemberCommandResponse>>
+    {
+        private readonly IApprenticesWriteRepository _apprenticesWriteRepository;
+        private readonly IApprenticesReadRepository _apprenticesReadRepository;
+        private readonly IAuditWriteRepository _auditWriteRepository;
+        private readonly IAanDataContext _aanDataContext;
+        private readonly ILogger<PatchApprenticeMemberCommandHandler> _logger;
+
+        public PatchApprenticeMemberCommandHandler(IApprenticesWriteRepository apprenticesWriteRepository, 
+            IApprenticesReadRepository apprenticesReadRepository, 
+            IAuditWriteRepository auditWriteRepository,
+            IAanDataContext aanDataContext,
+            ILogger<PatchApprenticeMemberCommandHandler> logger)
+        {
+            _apprenticesWriteRepository = apprenticesWriteRepository;
+            _apprenticesReadRepository = apprenticesReadRepository;
+            _auditWriteRepository = auditWriteRepository;
+            _aanDataContext = aanDataContext;
+            _logger = logger;
+        }
+
+        public async Task<ValidatableResponse<PatchApprenticeMemberCommandResponse>> Handle(PatchApprenticeMemberCommand command, CancellationToken cancellationToken)
+        {
+            var apprentice = await
+                _apprenticesReadRepository.GetApprenticeByApprenticeIdAndUserId(command.ApprenticeId, command.RequestedByMemberId);
+
+            if (apprentice == null)
+            {
+                _logger.LogError("Apprentice not found for ApprenticeId: {apprenticeid} and UserId: {userid}", command.ApprenticeId, command.RequestedByMemberId);
```

Comment:
It's pedantic but `{apprenticeid}` should be `{apprenticeId}` :D

Reply:
`apprenticeid` has now been updated to `apprenticeId`